### PR TITLE
Add const Uuid::from_uuid_bytes

### PR DESCRIPTION
**Pull request by mokeyish**
**I'm submitting a  const Uuid::from_uuid_bytes for creating const uuid value.**  [ref uuid_macro](https://github.com/uuid-rs/uuid_macro/pull/1)
  - [ ] bug fix
  - [x] feature enhancement
  - [ ] deprecation or removal
  - [ ] refactor

# Description

# Motivation

# Tests

# Related Issue(s)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -111,6 +111,7 @@
 )]
 #![deny(warnings)]
 #![cfg_attr(not(feature = "std"), no_std)]
+#![feature(const_fn)]
```
Comment (Dylan-DPC) [inline src/lib.rs]: this is not allowed for the stable build. you will have to wrap it in a `cfg_if`

File: src/lib.rs [hunk 2]
```diff
@@ -111,7 +111,8 @@
 )]
 #![deny(warnings)]
 #![cfg_attr(not(feature = "std"), no_std)]
-#![feature(const_fn)]
+#![cfg_attr(feature = "nightly", feature(const_fn))]
```
Comment (kinggoesgaming) [inline src/lib.rs]: write this as `#![cfg_attr(nightly, feature(const_fn))]`

File: Cargo.toml [hunk 3]
```diff
@@ -33,6 +33,7 @@ serde_test = "1.0.19"
 
 [features]
 default = ["std"]
+nightly = []
```
Comment (kinggoesgaming) [inline Cargo.toml]: I don't think we need this for this atm.. see the other comment below

Comment (mokeyish) [reply Cargo.toml]: @kinggoesgaming It is not working when I remove ```nightly = []``` and use ```#![cfg_attr(nightly, feature(const_fn))]```

Comment (kinggoesgaming) [reply Cargo.toml]: tested it locally.. its works for me... have you tried `cargo +nightly build`? it needs that `+nightly` bit... if you get something like toolchain not found error... run `rustup install nightly` and try again

Comment (Dylan-DPC) [reply Cargo.toml]: travis passes as well on nightly means it is working

Comment (mokeyish) [reply Cargo.toml]: @kinggoesgaming @Dylan-DPC Why is it only ```test_uuid_macro_build```test passed? The ```test_const_uuid_macro_build``` should also be passed.
image

Comment (kinggoesgaming) [reply Cargo.toml]: hmm its not even being detected/run, which is indeed weird... 

have you pushed these changes to your fork?

Comment (mokeyish) [reply Cargo.toml]: @kinggoesgaming It's didn't work as expectedly, So I haven't pushed it yet.

Comment (mokeyish) [reply Cargo.toml]: @kinggoesgaming I pushed it on [uuid_macro/nightly-uuid](https://github.com/uuid-rs/uuid_macro/tree/nightly-uuid)

File: src/lib.rs [hunk 4]
```diff
@@ -714,10 +716,16 @@ impl Uuid {
     ///
     /// let uuid = Uuid::from_uuid_bytes(bytes);
     /// ```
+    #[cfg(not(feature = "nightly"))]
     pub fn from_uuid_bytes(b: UuidBytes) -> Uuid {
         Uuid { bytes: b }
     }
 
+    #[cfg(feature = "nightly")]
```
[the remark this answers is not in the document]
Comment (mokeyish) [reply src/lib.rs]: @kinggoesgaming ```#[cfg(nightly)]``` didn't working on  [here](https://play.rust-lang.org/?gist=5a15b198e0472b46383e47a24ca01259&version=nightly&mode=debug)

File: Cargo.toml (hunk 3 above)
Comment (Dylan-DPC) [reply Cargo.toml]: based on the discussion in #220 we can go back to this. so this is fine

File: src/lib.rs (hunk 4 above)
[the remark this answers is not in the document]
Comment (Dylan-DPC) [reply src/lib.rs]: based on the discussion in #220 we can go back to this. so this is fine

**Pull request merged**